Tools/CISupport/ews-build/steps.py

```diff
@@ -3308,6 +3304,7 @@ class RunWebKitTests(shell.Test, AddToLogMixin):
     test_failures_log_name = 'test-failures'
     results_db_log_name = 'results-db'
     ENABLE_GUARD_MALLOC = False
+    ENABLE_ADDITIONAL_ARGUMENTS = True
```

Comment:
This (and other changes in RunWebKitTests and RunWebKitTestsInStressMode) are not strictly needed to fix the issue. But these are code refactoring to adopt cleaner approach to handle additionalArguments in RunWebKitTestsInStressMode.